modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportExport.testcase

```diff
@@ -27,7 +27,45 @@ definition {
 
 			JSONUser.tearDownNonAdminUsers();
 
-			ObjectAdmin.deleteObjectViaAPI(objectName = "Coupon");
+			ObjectAdmin.deleteAllCustomObjectsViaAPI();
+		}
+	}
+
+	@priority = 4
+	test CanDownloadJsonTFormatFileAsJson {
+		property portal.acceptance = "true";
+
+		task ("Given in Import/Export center") {
+			BatchEngine.setUpGlobalIds();
+
+			ImportExport.openImportExportAdmin();
+		}
+
+		task ("When Exporting File with JSONT format") {
+			ImportExport.exportFile(
+				entityType = "Account (v1_0 - Liferay Headless Admin User)",
+				exportFields = "name",
+				exportFileFormat = "JSONT");
+		}
+
+		task ("Then the downloaded zip file contain *.json file") {
+			DownloadTempFile();
+
+			var filePath = TestCase.getDependenciesDirPath(fileName = "account_import_jsont.json");
```

Comment:
The file we upload into dependencies folder is .txt, seems we'd like to use .json here

Reply:
I'm uploading it as .txt cause if I upload it as json it gets formatted and then later on the comparison fails because of added with formatting spaces. And yeah, test should be ignored, I just wanted to make sure what is happening on CI if we run it.